client/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:14.16.1-alpine3.13
+LABEL maintainer="hello@vizzuality.com"
+
+ARG UID=5000
+ARG GID=5000
+
+ENV NAME landgriffon-client
+ENV USER $NAME
+ENV APP_HOME /opt/$NAME
+
+RUN addgroup -g $GID $USER &&  adduser -u $UID -D -G $USER $USER
+
+WORKDIR $APP_HOME
+RUN chown $USER:$USER $APP_HOME
+
+USER $USER
+
+COPY --chown=$USER:$USER package.json yarn.lock ./
+RUN yarn install --frozen-lockfile
+
+# NextJS project folders
+COPY --chown=$USER:$USER .next .storybook components containers cypress docs lib pages public services styles svgs ./
+# NextJS config files
+COPY --chown=$USER:$USER next.config.js next-env.d.ts now.json postcss.config.js tailwind.config.js ./
+# Project config giles
+COPY --chown=$USER:$USER .* ./
```

Comment:
I believe we can avoid passing linting and development related config stuff to the container, but wasn't sure either.

Reply:
you can, assuming you don't want to run the tests inside the container (we I think we don't need to). @davidsingal thoughts on this? Are we missing anything needed for the build process?

Reply:
the BE image has the test stuff, so maybe add it here too...?

Reply:
I'm adding all the testing stuff so it can run inside the container (just in case) but I was referring mostly to stuff like eslint conf, which (i think) shouldn't be necessary inside docker ecosystem

Reply:
I was adding cypress folder as part of NextJS folders copy and I was forgetting the config file.
Thanks for pointing out. Changes -> b17c4bfc1e158409c45467696150de180caaa457

Reply:
I don't think we need run tests inside the container. Will it help you if I move everything inside a `src` folder?
For instance, the folders `.storybook`, `.next` or `docs` are not needed.

Reply:
@davidsingal thanks for your reply. As @tiagojsag stated, he has included tests stuff in the BE image, so IMO we either include tests stuff on the FE image, or remove it from BE container, up to you guys, but if they are **not needed**, I'd go for the latter.

Besides that @davidsingal, if you can kindly provide me with a list of the stuff you need in order to run the container, I'll make sure to include just that. No need to change your file tree bc this 😃